src/main/java/org/killbill/billing/client/KillBillHttpClient.java

```diff
@@ -325,7 +325,7 @@ public Response doOptions(final String uri, final Multimap<String, String> optio
     }
 
     private <T> T doPrepareRequestAndMaybeFollowLocation(final String verb, final String uri, final Object body, final Multimap<String, String> optionsRo, final Multimap<String, String> optionsForFollow, final int timeoutSec, final Class<T> clazz, final boolean followLocation) throws KillBillClientException {
-        final Multimap<String, String> options = HashMultimap.<String, String>create(optionsRo);
+        final Multimap<String, String> options = HashMultimap.create(optionsRo);
```

Comment:
Reverted the changes. I think the type inference was incorporated in newer version of the 6 compiler. In theory, Java 6 indeed doesn't support this.

Out of curiosity, was that an issue with compiling the client? or running it? Regardless of the syntax used here, the generated byte code is the same.